CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+### v6.0.0 (2022-07-28)
+
+* **BREAKING** Removed support for Node 12.
+
+  The minimum supported version is now Node v14. For further information on our support policy, see: https://docs.newrelic.com/docs/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.
```

Comment:
Just "Node 14". It's cleaner.

Reply:
well, this has been going out in every changelog and this is the last. also historically this is how we've done it.  🤷🏻